Doppler.PushContact/Services/PushContactService.cs

```diff
@@ -484,9 +484,16 @@ public async Task<MessageDeliveryResult> GetDeliveredMessageSummarizationAsync(s
                     })
                     .ToListAsync();
 
-                var delivered = historyEventsResultFiltered.FirstOrDefault().GetValue("delivered", 0).AsInt32;
-                var notDelivered = historyEventsResultFiltered.FirstOrDefault().GetValue("notDelivered", 0).AsInt32;
-                var sent = delivered + notDelivered;
+                var delivered = 0;
+                var notDelivered = 0;
+                var sent = 0;
+
+                if (historyEventsResultFiltered.Count > 0)
+                {
+                    delivered = historyEventsResultFiltered.FirstOrDefault().GetValue("delivered", 0).AsInt32;
+                    notDelivered = historyEventsResultFiltered.FirstOrDefault().GetValue("notDelivered", 0).AsInt32;
+                    sent = delivered + notDelivered;
+                }
```

Comment:
Very, very, very minor.

But, in general, we try to set the values only one time in order to allow the compiler to let us know if we miss something, for example:

```suggestion
                int delivered;
                int notDelivered;
                int sent;

                if (historyEventsResultFiltered.Count > 0)
                {
                    delivered = historyEventsResultFiltered.FirstOrDefault().GetValue("delivered", 0).AsInt32;
                    notDelivered = historyEventsResultFiltered.FirstOrDefault().GetValue("notDelivered", 0).AsInt32;
                    sent = delivered + notDelivered;
                }
                else
                {
                    delivered = 0;
                    notDelivered = 0;
                    sent = 0;
                }
```

Relacionado: https://andresmoschini.github.io/desarmando-doppler/#p54